modules/sqids_bazel/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "homepage": "https://sqids.org/bazel",
+    "maintainers": [],
```

Comment:
Would you be interested in adding yourself as a maintainer for this bcr module?

Reply:
done :+1: